zulipterminal/model.py

```diff
@@ -1383,18 +1383,19 @@ def _handle_typing_event(self, event: Event) -> None:
         if (
             len(narrow) == 1
             and narrow[0][0] == "pm_with"
-            and sender_email in narrow[0][1].split(",")
+            and sender_email in [email.strip() for email in narrow[0][1].split(",")]
```

Comment:
Is this change a bugfix? Why are we stripping?

Reply:
Yep. Earlier only the first user was detected as there was a space after the comma. This problem was mentioned in the issue as well ("In a huddle/group PM, we only handle typing events from the first email in the narrow specification.")